### DISCUSSION TITLE
SMPlayer 23.12.0 on Ubuntu 24.04.1 Unable to Navigate DVDs

### DISCUSSION THREAD

**Discussion by smplayeruser** [smplayer-dev/smplayer#1105] [Q&A]
Thanks for smplayer. You made mpv usable and the video quality is better than VLC.

There are no DVD titles displayed with smplayer 23.12.0 (revision 10207) using Qt 5.15.13 (compiled with Qt 5.15.12) using MPV 0.37.0 running on Ubuntu 24.04.1.

When playing a DVD, Browse -> Title only displays <empty> in a very faint font.

It's possible to play different DVD titles by launching smplayer from the terminal. For example:

smplayer dvd://5

Will play the sixth title on the DVD.

My Questions

Is there a way to get smplayer to display the different titles on a DVD on Ubuntu?

Using the terminal, how do you play a particular title on a DVD copied to an external drive?

The version of smplayer for Xenial did everything I ever wanted it to do – a great media player. I could select and play any title on a DVD or a DVD copied to an external drive without using the terminal. What version of smplayer was this? Can I run that old version on 24.04.1? Where can I find it?

Thanks for your help.

**Comment by smplayer-dev** [comment]
Check the latest version, it has some fixes regarding the browse -> title menu.
https://github.com/smplayer-dev/smplayer/releases/tag/v24.5.0.10342

**Reply by smplayeruser** [reply]
Thanks smplayer-dev,

I just installed smplayer 24.5.0.10342 with Gdebi.

It navigates DVDs with Browse -> Title without a problem. It even displays the run times. I don't need to use the terminal anymore.

Unfortunately, I can't find out how to play DVDs saved to an external drive - a folder with an AUDIO_TS and a VIDEO_TS in it. Previous versions had an option to play a DVD saved to a folder under either Open -> Disk or Open -> File. Sorry, I can't remember and I unistalled 23.12.0.

This worked perfectly on the version of smplayer for Xenial. Attempting this on smplayer for Jammy Jellyfish failed completely. Smplayer 23.12.0 on Ubuntu 24.04.1 could play the first title of a DVD saved to an external drive. I couldn't find out how to play the other titles. I can't remember if it worked on Beaver.

So how do I play and navigate DVDs saved to an external drive?

Once again, the old version of smplayer for Xenial did everything perfectly. What version of smplayer was this? Can I run that old version on Ubuntu 24.04.1? Where can I find it?

Thanks again for your help.

**Reply by smplayeruser** [reply]
Sorry to bother you again. I'm an idiot. I just found it under Open -> Directory.

It works great. It's like reconnecting with an old friend.

Was this always under Open -> Directory?

Thanks again.
